coraza.conf-recommended

```diff
@@ -35,22 +35,25 @@ SecRule REQUEST_HEADERS:Content-Type "^application/[a-z0-9.-]+[+]json" \
      "id:'200006',phase:1,t:none,t:lowercase,pass,nolog,ctl:requestBodyProcessor=JSON"
 
 # Maximum request body size we will accept for buffering. If you support
-# file uploads then the value given on the first line has to be as large
-# as the largest file you are willing to accept. The second value refers
-# to the size of data, with files excluded. You want to keep that value as
-# low as practical.
-#
+# file uploads, this value must has to be as large as the largest file
+# you are willing to accept.
 SecRequestBodyLimit 13107200
 
+# Maximum request body size that Coraza will store in memory. If the body
+# size exceeds this value, it will be saved to a temporary file on disk.
 SecRequestBodyInMemoryLimit 131072
 
-# SecRequestBodyNoFilesLimit is currently not supported by Coraza
+# Maximum request body size we will accept for buffering, with files excluded.
+# You want to keep that value as low as practical.
+# Note: SecRequestBodyNoFilesLimit is currently NOT supported by Coraza
```

Comment:
- `first line`, `second value` is not clear at all. I think it was referring to `SecRequestBodyNoFilesLimit` but then we added `SecRequestBodyInMemoryLimit` in between. I split this comment into one comment per directive.
- `SecRequestBodyInMemoryLimit` was uncommented